blocks/block-helpers.js

```diff
@@ -0,0 +1,27 @@
+/**
```

Comment:
I've added block-helpers.js for reusable functions under /blocks. Let me know what you think.